test/test_results.py

```
@@ -161,7 +161,7 @@ def test_get_result_hash_from_prefix(tmpdir):
     assert 'one of multiple commits' in str(excinfo.value)
 
 
-def test_backward_compat_load(example_results): # noqa uses fixture
+def test_backward_compat_load(example_results): # noqa F811
```

Can you add the comment on why we are ignoring this error please